Resources/config/tree.xml

```diff
@@ -4,10 +4,8 @@
         <service id="sonata.admin.doctrine_phpcr.tree_controller" class="Sonata\DoctrinePHPCRAdminBundle\Controller\TreeController">
             <argument type="service" id="doctrine_phpcr"/>
             <argument>%doctrine_phpcr.default_session%</argument>
-            <argument>%sonata_admin_doctrine_phpcr.tree_block.repository_name%</argument>
+            <argument>%sonata_admin_doctrine_phpcr.tree_block.configuration%%</argument>
```

Comment:
I think we should add WebTests at all to test the controllers and internals. So i would suggest: #457